src/proj.h

```diff
@@ -328,6 +340,17 @@ extern const PJ      *proj_shutdown;
 #endif
 
 
+/* Set or read error level */
+int  proj_errno (PJ *P);
+void proj_errno_set (PJ *P, int err);
```

Comment:
"""They are quite useful for reporting from application specific functions, that take PJ arguments.""" Hum, given that proj errno numbers are an internal detail, I don't see how user applications could know which code to use.

I'm just fighting to keep proj.h at the very strict minimum to reduce the API surface.

"pj_ctx_set_errno() is also part of proj_api.h - I assume for the same reason" : I'm pretty sure this wasn't really intended to be used by applications, but just because there is no proj internal include file. That should be there.